react/AppIcon/Readme.md

```diff
@@ -33,46 +34,43 @@ Is it also possible to provide a custom asynchronous `fetchIcon` which takes an
 ```
 
 ```jsx
-import AppIcon from 'cozy-ui/transpiled/react/AppIcon';
+import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
 const fetchIcon1 = () => 'https://placeholder.pics/svg/100/7DC4FF/Test%20Icon'
 const fetchIcon2 = () => 'https://placeholder.pics/svg/100/FF0202/Test%20Icon'
 
 const handleSwitch = () => {
-  const newFetchIcon = state.fetchIcon.name === 'fetchIcon1' ? fetchIcon2 : fetchIcon1
-  setState({ fetchIcon: newFetchIcon  })
+  const newFetchIcon =
+    state.fetchIcon.name === 'fetchIcon1' ? fetchIcon2 : fetchIcon1
+  setState({ fetchIcon: newFetchIcon })
 }
 
-initialState = { fetchIcon: fetchIcon1 };
-
-<div>
-  <button onClick={handleSwitch}>Switch</button>{' '}
-  fetchIcon: {state.fetchIcon.name}<br/>
+initialState = { fetchIcon: fetchIcon1 }
+;<div>
+  <button onClick={handleSwitch}>Switch</button> fetchIcon:{' '}
+  {state.fetchIcon.name}
+  <br />
```

Comment:
It seems the JSX code here went into a linter, it would be easier to see the real changs of the PR if these changes were in another commit.

Reply:
Hmm yeah there are some actual changes that i want in there but I didn't realize the linter went overboard. I'll try to re-edit that file.